frontend/src/utils.js

```diff
@@ -413,8 +413,11 @@ export const formatDate = (dateString) => {
   return date.toLocaleString("fr", options)
 }
 
-export const sectorsSelectList = (sectors) => {
+export const sectorsSelectList = (sectors, category) => {
```

Comment:
```suggestion
export const sectorsSelectList = (sectors, category = null) => {
```
Petit détail : peut-être en ajoutant une valeur par défaut c'est plus clair que `category` n'est pas obligatoire en appelant cette fonction.